### ISSUE TITLE
Remove deprecated functions in `lore/tree.py`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5127]
|<img alt="facundobatista's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| facundobatista reported|
|-|-|
|Trac ID|trac#5127|
|Type|task|
|Created|2011-06-01 02:38:12Z|
|Branch|https://github.com/twisted/twisted/tree/lore-tree-compare-5127|

This patch remove `compareMarkPos` and `comparePosition` from `lore/tree.py`, as they were deprecated for v9.

Attachments:

* [remove-lore-tree-deprecated-a.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/639/639520ebf6f1d416ab86828dc373b4ec290626ed/b392b3600f9a29b4edb844f0219980ffac3baa60.patch) (2529 bytes) - added by facundobatista on 2011-06-01 02:38:25Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5127 5127
type__task task
reporter__facundobatista facundobatista
priority__normal normal
milestone__ 
branch__branches_lore_tree_compare_5127 branches/lore-tree-compare-5127
branch_author__facundobatista facundobatista
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__ 
time__1306895892000000 1306895892000000
changetime__1308250548000000 1308250548000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [32110]) Branching to 'lore-tree-compare-5127'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [32111]) Apply remove-lore-tree-deprecated-a.patch, add news file. refs [#5127](https://github.com/twisted/twisted/issues/5127)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

The [doc build](http://buildbot.twistedmatrix.com/builders/documentation/builds/1332) succeeded after this change (isnt that all that matters ;). Branch up for review.

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli commented|
|-|-|

(In [32125]) Removed the unused twisted versions import as well

refs [#5127](https://github.com/twisted/twisted/issues/5127)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_6"></a>|@cyli commented|
|-|-|

[build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/lore-tree-compare-5127) look good.  Merging.  Thanks for applying the patch thijs!

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_7"></a>|@cyli set status to `closed`|
|-|-|

(In [32126]) Merge lore-tree-compare-5127: Remove deprecated compare methods in lore

Author: facundobatista
Reviewer: cyli
Fixes: [#5127](https://github.com/twisted/twisted/issues/5127)

Removes twisted.lore.tree.compareMarkPos and twisted.lore.tree.comparePosition, both of which were deprecated in v9.0.
